generator.yaml

```diff
@@ -17,15 +21,6 @@ resources:
       Code:
         compare:
           is_ignored: true
-      Layers:
-        set:
-          - method: Create
-            ignore: true
-      LayerStatuses:
-        is_read_only: true
-        from:
-          operation: GetFunction
-          path: Configuration.Layers
     renames:
       operations:
         CreateFunction:
```

Comment:
these deletions (and the ones below) in the generator.yaml file indicate functional changes to the Lambda controller's CRDs and controller implementation that are unrelated to the v0.19.2 runtime upgrade... can you explain?